numba/tests/test_looplifting.py

```diff
@@ -70,6 +70,10 @@ def lift_gen1(x):
         a[i] = x
     yield np.sum(a)
 
+def lift_issue2561(x):
+    for i in range(10):
+        for j in range(10):
+            return
```

Comment:
If we added and check the return value, we will find that the looplifting is not handling the `return` statement properly.